service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storage/CreateAzureStorageStep.java

```diff
@@ -11,25 +11,37 @@
 import bio.terra.workspace.service.resource.controlled.cloud.azure.storage.resourcemanager.data.CreateStorageAccountRequestData;
 import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.ControlledResourceKeys;
 import bio.terra.workspace.service.workspace.model.AzureCloudContext;
+import com.azure.core.http.HttpClient;
 import com.azure.core.management.Region;
 import com.azure.core.management.exception.ManagementException;
 import com.azure.resourcemanager.storage.StorageManager;
+import com.azure.resourcemanager.storage.models.StorageAccount;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.models.BlobCorsRule;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class CreateAzureStorageStep implements Step {
   private static final Logger logger = LoggerFactory.getLogger(CreateAzureStorageStep.class);
+  private static final String CORS_ALLOWED_METHODS = "GET,HEAD,OPTIONS,PUT,PATCH,POST,MERGE,DELETE";
```

Comment:
We can config these values out if need be, but I don't expect them to vary per-environment.

Reply:
I don't see a ton of room for that to expand anyway